File: Max-master/Max.java
import java.util.Scanner;

public class Max {

    public static void main(String[] args) {
        // variables and scanner object
        int num1, num2, num3;
        Scanner input = new Scanner(System.in);
        
        // prompt user for three numbers
        System.out.print("Enter a number: ");
        num1 = input.nextInt();
        
        System.out.print("Enter a second number: ");
        num2 = input.nextInt();
        
        System.out.print("Enter a third number: ");
        num3 = input.nextInt();
        
        // find the largest number and print it out
        if (num1 > num2 && num1 > num3){
            System.out.print(num1+"is the largest number.\n");
            System.exit(0);
        }
        else if (num2 > num3){
            System.out.print(num2+" is the largest number.\n");
            System.exit(0);
        }
        else{
            System.out.print(num3+" is the largest number.\n");
            System.exit(0);
        }
    }
    
}

File: Max-master/Max.c
#include <stdio.h>

int main()
{
    int num1, num2, num3;

    // prompt user for three numbers
    printf("Enter a number: ");
    scanf("%d",&num1);

    printf("Enter a second number: ");
    scanf("%d",&num2);

    printf("Enter a third number: ");
    scanf("%d",&num3);

    // find which number is the greatest and return
    if (num1 > num2 && num1 > num3){
        printf("%d is the greatest number.", num1);
        return 0;
    }
    else if (num2 > num3){
        printf("%d is the greatest number.", num2);
        return 0;
    }
    else{
        printf("%d is the greatest number.", num3);
        return 0;
    }

}
